packages/flex-dev-utils/src/__tests__/credentials.test.ts

```diff
@@ -0,0 +1,236 @@
+import * as credentials from '../credentials';
+
+jest.mock('keytar');
+jest.mock('../logger');
+jest.mock('../inquirer');
+
+// tslint:disable
+const keytar = require('keytar');
+const inquirer = require('../inquirer');
+// tslint:enable
+
+describe('credentials', () => {
+  // @ts-ignore
+  const exit = jest.spyOn(process, 'exit').mockReturnValue();
+
+  const accountSid = 'AC00000000000000000000000000000000';
+  const authToken = 'abc123';
+  const credential = {
+    account: accountSid,
+    password: authToken,
+  };
+
+  const OLD_ENV = process.env;
+
+  beforeEach(() => {
```

Comment:
would recommend `afterEach`

Reply:
all tests are using `beforeEach`; I'll fix them all in one PR and make sure I don't run into any issues; will also attempt to remove unnecessary calls to `.mockRestore()` if I can

Reply:
👍